frontend/src/app/modules/reporting/reporting-page/reporting-page.component.ts

```diff
@@ -1,7 +1,15 @@
 import {Component, OnInit, ViewEncapsulation} from "@angular/core";
+import {registerTableSorter} from "core-app/modules/reporting/reporting-page/functionality/tablesorter";
 
 export const reportingPageComponentSelector = 'op-reporting-page';
 
+import './functionality/reporting_engine';
+import './functionality/reporting_engine/filters';
+import './functionality/reporting_engine/group_bys';
+import './functionality/reporting_engine/restore_query';
+import './functionality/reporting_engine/controls';
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/7c43ca799794047bcf272d9ecda248b84d763c76/frontend/src/app/modules/reporting/reporting-page/reporting-page.component.ts#L12"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>